buildSrc/src/main/resources/plugin-descriptor.properties

```diff
@@ -43,3 +43,12 @@ extended.plugins=${extendedPlugins}
 #
 # 'has.native.controller': whether or not the plugin has a native controller
 has.native.controller=${hasNativeController}
+#
+# 'type': the type of this plugin. 'isolated' indicated a typical sandboxed plugin,
+# whereas 'bootstrap' indicates a plugin whose jars are added to the JVM's boot
+# classpath.
+type=${type}
+#
+# 'java.opts': any additional command line parameters to pass to the JVM when
+# Elasticsearch starts. Only applies when 'type' is 'bootstrap'.
+<%= type == "bootstrap" ? "" : "# " %>java.opts=${javaOpts}
```

Comment:
It is. Perhaps it's overkill, but I thought it was worth some extra discouragement from anyone setting this field. I can remove it if you feel it's too much.

Reply:
I'm not understand why it would be commented. If it is a bootstrap plugin, the java.opts should be added. If it is an isolated plugin, then why add java.opts at all, commented or otherwise?